xkit.js

```diff
@@ -1015,7 +964,7 @@ XKit = {
 
 
 function xerror(message) {
-	XKit.console.add(message);
+	console.log(message);
 	alert("XKit Error:\n" + message + "\n\nPlease refresh the page and try again or file a bug report at xkit.info.");
```

Comment:
Can you update this to point to new-xkit-support.tumblr.com/ask? Not sure if anything uses it anymore though so maybe not worth it